target/linux/ath79/dts/qca9533_qca_ap143.dts

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "qca,ap143", "qca,qca9533";
+	model = "Qualcomm Atheros AP143 reference board";
+
+	aliases {
+		label-mac-device = &wmac;
```

Comment:
This does not work. It can (typically) only be used in combination with mtd-mac-address.
Use label_mac in board.d/02_network instead (you will find examples there), or use &eth0/&eth1 here instead if it happens to have the same address as wmac.

Reply:
Indeed - I've dropped this for now.